app/components/post_list/post_list.js

```diff
@@ -76,9 +76,16 @@ export default class PostList extends PureComponent {
     scrollList = () => {
         InteractionManager.runAfterInteractions(() => {
             if (this.props.postIds.length && this.newMessagesIndex !== -1) {
-                this.refs.list.scrollToIndex({index: this.newMessagesIndex, viewPosition: 1, viewOffset: -10, animated: true});
+                if (this.refs.list) {
```

Comment:
If there's any Jira tickets for these, can you add a link back to this PR so we know that they're fixed?

Reply:
I couldn't find one for this